cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtilsTest.java

```diff
@@ -0,0 +1,135 @@
+    Assert.assertEquals(Long.valueOf(START_TIME_STRING), startMs);
+    Assert.assertEquals(Long.valueOf(END_TIME_STRING), endMs);
+    ParameterUtils.validateTimeRange(startMs, endMs);
+
+    startMs = ParameterUtils.startMsOrDefault(mockRequest, DEFAULT_START_TIME_MS);
+    endMs = ParameterUtils.endMsOrDefault(mockRequest, DEFAULT_END_TIME_MS);
+
+    Assert.assertEquals(Long.valueOf(START_TIME_STRING), startMs);
+    Assert.assertEquals(Long.valueOf(END_TIME_STRING), endMs);
+  }
+
+  @Test
+  public void testParseTimeRangeNotSet() {
+    HttpServletRequest mockRequest = EasyMock.mock(HttpServletRequest.class);
+    // Mock the request so that it does not contain start/end time
+    EasyMock.expect(mockRequest.getParameterMap()).andReturn(Collections.emptyMap()).times(4);
+    EasyMock.replay(mockRequest);
+
+    Long startMs = ParameterUtils.startMsOrDefault(mockRequest, null);
+    Long endMs = ParameterUtils.endMsOrDefault(mockRequest, null);
+    Assert.assertNull(startMs);
+    Assert.assertNull(endMs);
+
+    startMs = ParameterUtils.startMsOrDefault(mockRequest, DEFAULT_START_TIME_MS);
+    endMs = ParameterUtils.endMsOrDefault(mockRequest, DEFAULT_END_TIME_MS);
+
+    Assert.assertEquals(DEFAULT_START_TIME_MS, startMs);
+    Assert.assertEquals(DEFAULT_END_TIME_MS, endMs);
+  }
+
+  @Test
+  public void testParseReplicationThrottleWithNoDefault() {
+    HttpServletRequest mockRequest = EasyMock.mock(HttpServletRequest.class);
+    KafkaCruiseControlConfig controlConfig = EasyMock.mock(KafkaCruiseControlConfig.class);
+
+    Map<String, String[]> paramMap = new HashMap<>();
+    paramMap.put(ParameterUtils.REPLICATION_THROTTLE_PARAM, new String[]{ParameterUtils.REPLICATION_THROTTLE_PARAM});
+
+    EasyMock.expect(mockRequest.getParameterMap()).andReturn(paramMap).times(1);
+    EasyMock.expect(mockRequest.getParameter(ParameterUtils.REPLICATION_THROTTLE_PARAM)).andReturn(REPLICATION_THROTTLE_STRING).times(1);
+    EasyMock.expect(controlConfig.getLong(ExecutorConfig.DEFAULT_REPLICATION_THROTTLE_CONFIG)).andReturn(null); // No default
+
+    EasyMock.replay(mockRequest);
+    EasyMock.replay(controlConfig);
+
+    Long replicationThrottle = ParameterUtils.replicationThrottle(mockRequest, controlConfig);
+    Assert.assertEquals(Long.valueOf(REPLICATION_THROTTLE_STRING), replicationThrottle);
+  }
+
+  @Test
+  public void testParseReplicationThrottleWithDefault() {
+    HttpServletRequest mockRequest = EasyMock.mock(HttpServletRequest.class);
+    KafkaCruiseControlConfig controlConfig = EasyMock.mock(KafkaCruiseControlConfig.class);
+    // No parameter string value in the parameter map
+    EasyMock.expect(mockRequest.getParameterMap()).andReturn(Collections.emptyMap()).times(1);
+    EasyMock.expect(controlConfig.getLong(ExecutorConfig.DEFAULT_REPLICATION_THROTTLE_CONFIG))
+            .andReturn(Long.valueOf(DEFAULT_REPLICATION_THROTTLE_STRING));
+
+    EasyMock.replay(mockRequest);
+    EasyMock.replay(controlConfig);
```

Comment:
(There are other similar uses in this test):
This function takes `Object...` as input; hence, we can pass all mocks in a single call rather than sequential calls.